diff --git a/src/pgtkterm.c b/src/pgtkterm.c
index c00e135..7094f9c 100644
--- a/src/pgtkterm.c
+++ b/src/pgtkterm.c
@@ -6792,7 +6792,7 @@ pgtk_term_init (Lisp_Object display_name, char *resource_name)
 
   /* If the PGTK port is being used under X, complain very loudly, as
      that isn't supported.  */
-  pgtk_display_x_warning (dpy);
+  /* pgtk_display_x_warning (dpy); */
 
   dpyinfo = xzalloc (sizeof *dpyinfo);
   pgtk_initialize_display_info (dpyinfo);
